_includes/fellow-filters.html

```diff
@@ -11,6 +11,19 @@
         {% endfor %}
       </select>
     </div>
+    <div class="filters__left display-flex">
+      <div class='filter__col'>
+        <select class="usa-select" name="fellows-select-skill" id="fellows-select-skill">
+          <option value="" selected disabled hidden>Fellows by specialty</option>
+          {% assign skills = site.data.fellow-specialties | reverse %}
+          {% for skill in skills %}
+            <option value="{{ skill }}">
+              {{ skill }}
+            </option>
```

Comment:
Creating the dropdown of specialty skills.

Reply:
Could we add in "All Skills" option at the top of the list, so people could default back to the full list of fellows?

Reply:
@keithpif yep! That's on line 19 (It currently reads "all specialities" but that's easy to change to "skills" if you'd rather use that language).

Reply:
@hbillings I'm not seeing it in there, 

> @keithpif I believe that has something to do with how the Jekyll paginator element is set up. I'm out the rest of today but happy to look into this later if you like.

Yup, it is set in the config file. I'll change to 12!

Reply:
@hbillings I'm not seeing "all specialities", is it in this file?